drivers/aks/aks_driver.go

```diff
@@ -792,11 +792,6 @@ func (d *Driver) createOrUpdate(ctx context.Context, options *types.DriverOption
 			osDiskSizeGBPointer = to.Int32Ptr(int32(driverState.AgentOsdiskSizeGB))
 		}
 
-		agentDNSPrefix := driverState.AgentDNSPrefix
```

Comment:
there is no more dns prefix?

Reply:
Looks like it did get removed, https://github.com/rancher/kontainer-engine/blob/328f210777cdcbcd29475e9d058736d7c362d9c2/vendor/github.com/Azure/azure-sdk-for-go/services/containerservice/mgmt/2018-03-31/containerservice/models.go#L1080:6 

But, it's still present at cluster level https://github.com/rancher/kontainer-engine/blob/328f210777cdcbcd29475e9d058736d7c362d9c2/drivers/aks/aks_driver.go#L838

Reply:
reminds me, should we be removing it from our flags too? 
https://github.com/rancher/kontainer-engine/blob/328f210777cdcbcd29475e9d058736d7c362d9c2/drivers/aks/aks_driver.go#L431

Reply:
@kinarashah I think we should. The param got deprecated last year in their sdk (https://github.com/Azure/azure-sdk-for-go/commit/c54b932da61fac394be959bd9ab4da287e039680#diff-3cc4ae0a07e198ede58c0be8ee523998) , chances are that it is no longer respected by the cloud provider either.